.gitignore

```diff
@@ -26,6 +26,7 @@ mdHelp.html
 test/e2e/data_setup/uploaded_files/*.*
 package-lock.json
 entities.json
+lib/switchUserAccounts.html
```

Comment:
I wish we had a better place for this. If we want to continue this path, then the lib function will be full of .html and .js files.

I feel like `lib` might not be the best place for this, but I really cannot think of a better place for it so maybe it's fine. But can you please create a folder for this `switchUserAccounts.html` (with the same name) and then you can rename the HTML to `index.html`. Then the URL will be just `/lib/switchUserAccounts`

Reply:
I tried doing this and had issues with makefile.. Not sure exactly the error but some of the paths weren't working properly.